### PR TITLE
feat: kafka connector support string timestamp and partial insert

### PR DESCRIPTION
**Pull request by vagetablechicken**
In auto schema mode, add supports:
- string timestamp, e.g. `{"ts_col": "2024-03-26 00:00:00"}`
- auto schema partial insert, e.g. t1 has two columns `c1,c2`, we can just insert json `{"c1":1}`, c2 will be the default value
note: null, e.g. `{"c1":null}` will set c1 to NULL, notice that `null` can't be string with quote `"`.

https://openmldb.ai/download/kafka-connector/kafka-connect-jdbc-10.5.0-SNAPSHOT-0.8.5.tgz has been updated to this version.

### CODE REVIEW

File: docs/zh/integration/online_datasources/kafka_connector_demo.md [hunk 1]
```diff
@@ -6,9 +6,31 @@ Apache Kafka是一个事件流平台。它可以作为OpenMLDB的在线数据源
...
+
+此功能可独立使用，如果此项为空，则使用`table.name.format`规则。`table.name.format`的规则是，将`${topic}`替换为topic名字，默认配置为`${topic}`，即表和topic同名。更复杂的例子是，format配置为`kafka_${topic}`，topic名为t1的话，表名将为`kafka_t1`。因此，format配置只适用于表名固定或包含topic名的情况。
+
+## 性能
+
+Kafka利用OpenMLDB Kafka Connector导入数据到OpenMLDB集群，其性能将受Kafka发送端和OpenMLDB接收端共同影响。我们提供单机和集群两种情况下的性能测试报告，见[Kafka Perf Test](https://github.com/vagetablechicken/openmldb-compose?tab=readme-ov-file#performance-test)。
+
+假设topic的数据量足够多，Kafka导入效率基本由Kafka topic的分区数和Connector的task数决定，如果这两者不够大，写入到OpenMLDB的并发度就不够高。task数量较大时，单个Kafka Connect服务也会收到单机物理资源限制，此时需要部署分布式的Kafka Connect，将tasks均匀分散到多台机器上，提高并发度。
+
+观察报告的pqs和latency，单机情况下，机器CPU core 40较充足，Kafka Connector可以配置较多的task，OpenMLDB端的写入qps总计可以达到10w。但OpenMLDB内部的写入延迟并未明显提高，只是单机性能有限，使得写入表现如此。集群性能测试也证明了这一点，我们将OpenMLDB集群化后，单机部署Kafka Connect，Kafka topic分区和tasks数改变，qps不会有明显提高，而分布式部署Kafka Connect后，qps明显提高到了单台90k，总计180k。
```
Comment (vagetablechicken) [inline docs/zh/integration/online_datasources/kafka_connector_demo.md]: update 3 connect process

**Review by aceforeverd — APPROVED** [review]

File: docs/zh/integration/online_datasources/kafka_connector_demo.md [hunk 2]
```diff
@@ -6,9 +6,31 @@ Apache Kafka是一个事件流平台。它可以作为OpenMLDB的在线数据源
...
+
+此功能可独立使用，如果此项为空，则使用`table.name.format`规则。`table.name.format`的规则是，将`${topic}`替换为topic名字，默认配置为`${topic}`，即表和topic同名。更复杂的例子是，format配置为`kafka_${topic}`，topic名为t1的话，表名将为`kafka_t1`。因此，format配置只适用于表名固定或包含topic名的情况。
+
+## 性能
+
+Kafka利用OpenMLDB Kafka Connector导入数据到OpenMLDB集群，其性能将受Kafka发送端和OpenMLDB接收端共同影响。我们提供单机和集群两种情况下的性能测试报告，见[Kafka Perf Test](https://github.com/vagetablechicken/openmldb-compose?tab=readme-ov-file#performance-test)。
+
+假设topic的数据量足够多，Kafka导入效率基本由Kafka topic的分区数和Connector的task数决定，如果这两者不够大，写入到OpenMLDB的并发度就不够高。task数量较大时，单个Kafka Connect服务也会收到单机物理资源限制，此时需要部署分布式的Kafka Connect，将tasks均匀分散到多台机器上，提高并发度。
+
+观察报告的pqs和latency，单机情况下，机器CPU core 40较充足，Kafka Connector可以配置较多的task，OpenMLDB端的写入qps单台50k，总计100k。但OpenMLDB内部的写入延迟并未明显提高，只是单机性能有限，使得写入表现如此。集群性能测试也证明了这一点，我们将OpenMLDB集群化后，单机部署Kafka Connect，Kafka topic分区和tasks数改变，qps不会有明显提高，而分布式部署Kafka Connect后，qps明显提高。两个Kafka Connect均分78个sink task写入，TabletServer QPS可以达到单台90k，总计180k；三个Kafka Connect均分120个sink task写入，TabletServer QPS可以达到单台100k，总计200k。
```
Comment (Elliezza) [inline docs/zh/integration/online_datasources/kafka_connector_demo.md]: QPS?

Comment (vagetablechicken) [reply docs/zh/integration/online_datasources/kafka_connector_demo.md]: OK

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Pull request merged**